presto-tests/src/test/java/io/prestosql/execution/TestEventListener.java

```diff
@@ -189,17 +191,20 @@ public void testKilledWhileWaitingForResources()
                     .build();
             String sql = format("SELECT nationkey as %s  FROM tpch.sf1.nation", testQueryMarker);
 
+            // preconfigure event collector filtering so helper queries are not recoreded
+            generatedEvents.reset(2, buildEventFilters(sql, true));
```

Comment:
i do not understand this comment. what would happen if we remove this code line?

Reply:
If we did not setup filtering in `generatedEvents` here we would register events coming from queries run as part of `executorService.submit`.